### PR TITLE
regenerate the client library surface

### PR DESCRIPTION
**Pull request by tangiel**
This is currently untested in production.

### CODE REVIEW

**Comment by tangiel** [comment]
Just tested on production and it seems to work.

File: endpoints_management/control/check_request.py [hunk 1]
```diff
@@ -143,7 +144,7 @@ def convert_response(check_response, project_id):
     api_key_is_bad: indicates that a given api_key is bad
 
     Args:
-       check_response (:class:`endpoints_management.gen.servicecontrol_v1_messages.CheckResponse`):
+       check_response (:class:`endpoints_management.gen.servicecontrol_v1_sc_messages.CheckResponse`):
```
Comment (inklesspen) [inline endpoints_management/control/check_request.py]: I don't think this change is quite correct.

File: endpoints_management/control/check_request.py [hunk 2]
```diff
@@ -168,13 +169,13 @@ def sign(check_request):
     """Obtains a signature for an operation in a `CheckRequest`
 
     Args:
-       op (:class:`endpoints_management.gen.servicecontrol_v1_messages.Operation`): an
+       op (:class:`endpoints_management.gen.servicecontrol_v1_sc_messages.Operation`): an
```
Comment (inklesspen) [inline endpoints_management/control/check_request.py]: Nor this one.

File: endpoints_management/control/check_request.py (hunk 1 above)
Comment (tangiel) [reply endpoints_management/control/check_request.py]: Done.

File: endpoints_management/control/check_request.py (hunk 2 above)
Comment (tangiel) [reply endpoints_management/control/check_request.py]: Done.

**Pull request merged**